ide/coqide/coqide.ml

```diff
@@ -1413,34 +1417,36 @@ let build_ui () =
   menu file_menu [
     item "File" ~label:"_File";
     item "New" ~callback:File.newfile ~stock:`NEW;
-    item "Open" ~callback:(File.load ~parent:w) ~stock:`OPEN;
+    item "Open" ~callback:(File.load ~parent:w) ~stock:`OPEN ~tooltip:"Open file";
     item "Save" ~callback:(File.save ~parent:w) ~stock:`SAVE ~tooltip:"Save current buffer";
-    item "Save as" ~label:"S_ave as" ~stock:`SAVE_AS ~callback:(File.saveas ~parent:w);
+    item "Save as" ~stock:`SAVE_AS ~callback:(File.saveas ~parent:w);
     item "Save all" ~label:"Sa_ve all" ~callback:File.saveall;
     item "Close buffer" ~label:"_Close buffer" ~stock:`CLOSE
       ~callback:(File.close_buffer ~parent:w) ~tooltip:"Close current buffer";
-    item "Print..." ~label:"_Print..."
-      ~callback:File.print ~stock:`PRINT ~accel:"<Ctrl>p";
-    item "Rehighlight" ~label:"Reh_ighlight" ~accel:"<Ctrl>l"
+    item "Print..." ~label:"_Print..." ~accel:"<Primary>p"
+      ~callback:File.print ~stock:`PRINT;
+    item "Rehighlight" ~label:"Re_highlight" ~accel:"<Primary>l"
       ~callback:File.highlight ~stock:`REFRESH;
     item "Quit" ~stock:`QUIT ~callback:(File.quit ~parent:w);
   ];
 
   menu export_menu [
-    item "Export to" ~label:"E_xport to";
-    item "Html" ~label:"_Html" ~callback:(File.export "html");
+    item "Export to" ~label:"E_xport script to";
+    item "Html" ~label:"_HTML" ~callback:(File.export "html");
     item "Latex" ~label:"_LaTeX" ~callback:(File.export "latex");
-    item "Dvi" ~label:"_Dvi" ~callback:(File.export "dvi");
-    item "Pdf" ~label:"_Pdf" ~callback:(File.export "pdf");
-    item "Ps" ~label:"_Ps" ~callback:(File.export "ps");
+    item "Dvi" ~label:"_DVI" ~callback:(File.export "dvi");
+    item "Pdf" ~label:"_PDF" ~callback:(File.export "pdf");
+    item "Ps" ~label:"P_ostScript" ~callback:(File.export "ps");
   ];
 
   menu edit_menu [
     item "Edit" ~label:"_Edit";
-    item "Undo" ~accel:"<Primary>u" ~stock:`UNDO
+    item "Undo" ~accel:"<Primary>z" ~stock:`UNDO
       ~callback:(cb_on_current_term (fun t -> t.script#undo ()));
-    item "Redo" ~stock:`REDO
+    item "Redo" ~accel:"<Primary><Shift>z" ~stock:`REDO
       ~callback:(cb_on_current_term (fun t -> t.script#redo ()));
+    item "Select All" ~accel:"<Primary>a" ~stock:`SELECT_ALL
+      ~callback:(cb_on_current_term (fun t -> t.script#select_all ()));
```

Comment:
This breaks selecting the contents of the Goal or Messages panel with CTRL A.

Reply:
Thank you. I think I was able to make it handle all the cases.